lib/mining/template.js

```diff
@@ -468,7 +445,7 @@ class BlockTemplate {
    */
 
   toCoinbase() {
-    return this.coinbase;
+    return this.coinbase.clone();
```

Comment:
Makes sense to return a clone here, but I can't find reference to this function - what's it used for?